workflow/tests/test_workflow_models.py

```diff
@@ -4,12 +4,92 @@
 from django.test import TestCase
 from workflow.models import (
     Organization, Program, Country, Province, ProjectAgreement, Sector,
-    ProjectComplete, ProjectType, SiteProfile, Office, Benchmarks, Budget
+    ProjectComplete, ProjectType, SiteProfile, Office, Benchmarks, Budget,
+    Documentation, Checklist, ProjectStatus
 )
 
+class ProjectStatusTestCase(TestCase):
+
+
+    def setUp(self):
+        new_project_status = ProjectStatus.objects.create(name="project status")
+        new_project_status.save()
```

Comment:
@mutebironald You don't need to call `.save()` after calling `.objects.create()`

https://stackoverflow.com/a/15726726/4017403

Reply:
check line 16, ```  new_project_status.save()```.